data/shaders/basic_vs.glsl

```diff
@@ -10,12 +11,11 @@ out vec4 v_Color;
 in vec4 i_World0;
 in vec4 i_World1;
 in vec4 i_World2;
-in vec4 i_World3;
 in vec4 i_Color;
 in vec4 i_UvRange;
 
 void main() {
-    mat4 m_World = mat4(i_World0, i_World1, i_World2, i_World3);
+    mat4 m_World = world_matrix(i_World0, i_World1, i_World2);
```

Comment:
can we also move the `in vec4 i_World?` into the header?

Reply:
If you think we need it, sure.